include/class.lock.php

```diff
@@ -90,6 +99,7 @@ function renew($lockTime=0) {
             SqlFunction::NOW(),
             SqlInterval::MINUTE($lockTime)
         );
+        $this->_signature = md5($this->lock_id.$this->code.strtotime($this->expire));
```

Comment:
Don't set hash in multiple places - getSignature should be the only spot.

```php
$this->_signature = null;
```

Reply:
@protich , do i have to implement only this $this->_signature = null; or this to $this->_signature = md5($this->lock_id.$this->code.strtotime($this->expire));.

Reply:
@jthakilla9

The changes are already implemented...just apply the changes from the Changes tab.

Cheers.

Reply:
Thanks, i will use the changes tab!